modules/oe_starter_content_event/config/install/field.field.node.oe_event.oe_featured_media.yml

```diff
@@ -0,0 +1,30 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_featured_media
+    - media.type.av_portal_photo
+    - media.type.image
```

Comment:
I think we don't really need this.

Reply:
Agreed we are. not exporting these configs.